viya_ark_library/k8s/sas_kubectl.py

```diff
@@ -99,6 +93,13 @@ def __init__(self, executable: Text = "kubectl", namespace: Text = None, global_
                            ):
                             self.ingress_ns = ns
                             break
+                else:
+                    ns: AnyStr = self.do("get namespace " + ingress_namespace, ignore_errors=True, warning=False)
+                    if type(ns) is str and _DO_MSG_ in ns:
+                        raise NamespaceNotFoundError(
+                            f"The ingress namespace [{ingress_namespace}] was not found in the target environment.")
```

Comment:
I think there needs to be a better way to handle this error, since it is not obvious why it should be causing a message at all, or what _DO_MSG_ has to do with it and makes it harder to maintain.

Reply:
code updated.